source/css/scss/app/pages/monthly-good.scss

```diff
@@ -149,7 +148,7 @@
 	}
 
 	.kiva-as-expert-section-wrapper {
-		background-color: $kiva-bg-lightgrey;
+		background-color: #fafafa;
```

Comment:
@BoulderBrains - not a reason not to merge, but wonder why you switched from the variable to a hard-coded color (since the variable is the value you had-coded)?